product_docs/docs/tde/15/secure_key/key_store/thales/thales_restapi.mdx

```diff
@@ -41,45 +41,45 @@ See [Installing EDB TDE Thales REST](/tde/latest/secure_key/key_store/thales/ins
 
 ## Creating a certificate for authentication
 
-Create a Certificate Signing Request (CSR) and then provide that CSR to a Certificate Authority (CA), so it can generate a certificate.
+Create a certificate signing request (CSR). Then provide that CSR to a certificate authority (CA) so it can generate a certificate.
 
 1.  Access your Thales CipherTrust Manager instance. 
 
-1.  Navigate to the **CSR tool** under **CA**:
+1.  Under **CA**, select **CSR Tool**.
 
     ![Navigate to CSR tool](./images/01_CSR_tool.png)
 
-1.  Create a CSR. Ensure you select the RSA option, `2048` size and then select **Create**: 
+1.  Create a CSR. Ensure you select the RSA algorithm and set the size to 2048. Select **Create**. 
 
     ![Create CSR](./images/02_CSR_creation.png)
 	
-1.  In the same pop-up window, after the certificate has been created, select **save csr** and **save private key** to download a `CSR.pem` and `key.pem` file.
+1.  To download a `CSR.pem` and `key.pem` file, after the certificate is created, in the same dialog box, select **save csr** and **save private key** .
 
 	![Download the CSR](./images/03_CSR_creation_download.png)
 
-1.  Provide a CSR to a certificate authority (CA) so it can generate a certificate. Navigate to **CA** > **Local**.
+1.  Provide a CSR to a CA so it can generate a certificate. Under **CA**, select **Local**.
 
     ![Navigate to local CA](./images/04_local_CA.png)
 
-1.  Select a CA. You can use the default local CA provided by CipherTrust. Then, select **Upload CSR**:
+1.  Select a CA. You can use the default local CA provided by CipherTrust. Then select **Upload CSR**.
 
     ![Upload CSR to CA](./images/05_local_CA_selected.png)
 
-1.  Enter the previously assigned **Display name**, paste the contents of the downloaded `CSR.pem` file and select **Issue Certificate**:
+1.  Enter the previously assigned **Display name**, paste the contents of the downloaded `CSR.pem` file, and select **Issue Certificate**:
     
 	![Enter CSR details to issue a certificate](./images/06_enter_pem_contents.png)
 
 1.  Download the `Certificate.pem` certificate: 
 
     ![Download the issued certificate](./images/07_download_certificate.png)
 
-1.  Update your user configuration to allow certificate-based login. Navigate to **Access Management** > **Users** and select your user.
+1.  Update your user configuration to allow certificate-based login. Under **Users**, select **Access Management and select your user.
```

Comment:
The other way around. Under **Access Management**, select **Users** and select your user.